web-components/src/components/icons/CoinGeckoIcon.tsx

```diff
@@ -0,0 +1,89 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+interface IconProps extends SvgIconProps {}
+
+export default function CoinGeckoIcon({
+  sx = [],
+  ...props
+}: IconProps): JSX.Element {
+  return (
+    <SvgIcon
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 1044.28 276.04"
+      sx={[{ fill: 'none' }, ...(Array.isArray(sx) ? sx : [sx])]}
+      {...props}
+    >
+      <defs>
+        <style>
+          {'.cls-1{fill:#fff}.cls-4{fill:#8bc53f}.cls-6{fill:#58595b}'}
+        </style>
+      </defs>
+      <title>{'coingecko_full vector (white font)'}</title>
```

Comment:
this title appears in the popover when hovering... maybe just `CoinGecko`

Reply:
Oh, this came from Figma export, did not see it. Will change it thanks.